cloudbuild.yaml

```diff
@@ -0,0 +1,7 @@
+steps:
```

Comment:
why is this file named `.yaml` and not `.yml` like the others?

Reply:
I followed the official namings here and this file is supposed to have the `.yaml` suffix.